Kitodo-DataManagement/pom.xml

```diff
@@ -142,9 +122,9 @@
             <version>${hibernate.version}</version>
         </dependency>
         <dependency>
-            <groupId>commons-lang</groupId>
-            <artifactId>commons-lang</artifactId>
-            <version>2.6</version>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.7</version>
```

Comment:
This library is used many times, please add a global version number property. This could be conflicting with needed changes on PR #1572 !